src/Agoda.IoC.Core/ContainerRegistrationExceptionContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Agoda.IoC.Core
+{
+    public class ContainerRegistrationContextException : Exception
+    {
+        public ContainerRegistrationContextException(string message): base(message)
+        {
+        }
+        public bool Ignore { get; set; }
+        public List<RegistrationContextException> RegistrationContextExceptions { get; } 
+             = new List<RegistrationContextException>();
+
+    }
+
+
+    public class RegistrationContextException : Exception
+    {
+        private readonly RegistrationContext _registrationContext;
```

Comment:
What is the point of this field? It doesn't seem to ever be consumed.

Reply:
I do need to see more information on this object.